xml/System.Net.Http/HttpClient.xml

```diff
@@ -79,29 +79,52 @@ On each platform, <xref:System.Net.Http.HttpClient> tries to use the best availa
-
+| Linux/.NET Core 1.0-2.0     | `libcurl`-based HTTP transport (HTTP 2.0 capable)                                         |
+| .NET Core 2.1 and later     | <xref:System.Net.Http.SocketsHttpHandler?displayProperty=nameWithType>                    |
 Users can also configure a specific transport for <xref:System.Net.Http.HttpClient> by invoking the <xref:System.Net.Http.HttpClient.%23ctor*> constructor that takes an <xref:System.Net.Http.HttpMessageHandler>.
 
 ### HttpClient and .NET Core
 
-Starting with .NET Core 2.1, the <xref:System.Net.Http.SocketsHttpHandler?displayProperty=nameWithType> class instead of `HttpClientHandler` provides the implementation used by higher-level HTTP networking classes such as `HttpClient`. You can configure your application to use the older <xref:System.Net.Http.HttpClientHandler?displayProperty=nameWithType> instead by calling the <xref:System.AppContext.SetSwitch%2A> method as follows:
+Starting with .NET Core 2.1, the <xref:System.Net.Http.SocketsHttpHandler?displayProperty=nameWithType> class instead of `HttpClientHandler` provides the implementation used by higher-level HTTP networking classes such as `HttpClient`. The use of <xref:System.Net.Http.SocketsHttpHandler> offers a number of advantages:
 
-```csharp
-AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", false);
-```
+- A significant performance improvement when compared with the previous implementation.
+
+- The elimination of platform dependencies, which simplifies deployment and servicing. For example, `libcurl` is no longer a dependency on .NET Core for macOS and .NET Core for Linux.
+
+- Consistent behavior across all .NET platforms.
+
+If this change is undesirable, you can configure your application to use the older <xref:System.Net.Http.HttpClientHandler?displayProperty=nameWithType> instead in a number of ways:
+
+- By calling the <xref:System.AppContext.SetSwitch%2A?displayProperty=nameWithType> method as follows:
 
-```vb
-AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", False)
-``` 
+   ```csharp
+   AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", false);
+   ```
+
+   ```vb
+   AppContext.SetSwitch("System.Net.Http.useSocketsHttpHandler", False)
+   ``` 
+- By defining the `System.Net.Http.useSocketsHttpHandler` switch in the *.netcore.runtimeconfig.json* configuration file:
+
+  ```json
+  "runtimeOptions": {
+    "configProperties": {
+        "System.Net.Http.useSocketsHttpHandler": false
+    }
+  }
+  ```
   
+- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
+
+- On Windows only, by passing an <xref:System.Net.Http.HttpClientHandler> instance to the <xref:System.Net.Http.HttpClient.%23ctor(System.Net.Http.HttpMessageHandler)?displayProperty=nameWithType> constructor to configure the `HTTPClient` on a per-instance basis.
```

Comment:
This makes it sound like the following code will use the old handler:
```csharp
client = new HttpClient(new HttpClientHandler());
```
Which isn't the case. HttpClientHandler used to be implemented on top of WinHttpHandler, but now it defaults to using SocketsHttpHandler unless the AppContextSwitch mentioned above is set.

Reply:
Yeah, I would strike this whole last sentence.